app/templating/schema_context.py

```diff
@@ -66,3 +69,49 @@ def json_and_html_safe(data):
     if isinstance(data, str):
         return escape(data.replace('\\', r'\\'))
     return data
+
+
+def _get_is_repeating_answer(answer_id):
+    return g.schema.is_repeating_answer_type(answer_id)
+
+
+def _get_answer_is_in_repeating_group(answer_id):
+    return g.schema.answer_is_in_repeating_group(answer_id)
+
+
+def _create_answers_list(answers, index_key):
+    """Creates a list of repeating answer values in the same index position
+    as that specified by their `group_instance` or `answer_instance`
+
+    Args:
+    :param answers: list of answer dicts
+    :param index_key: key of index value. 'group_instance' or 'answer_instance'
+
+    Returns:
+    List of values. Empty values are padded.
```

Comment:
Minor comment again your returning a list of values yet called the list 'items'. Would it not be better to call it 'values' instead for semantic purposes?